internal/cmd/base/delete.go

```diff
@@ -27,9 +28,9 @@ type DeleteCmd struct {
 // CobraCommand creates a command that can be registered with cobra.
 func (dc *DeleteCmd) CobraCommand(s state.State) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:                   fmt.Sprintf("delete [FLAGS] %s", strings.ToUpper(dc.ResourceNameSingular)),
+		Use:                   fmt.Sprintf("delete [FLAGS] %s...", strings.ToUpper(dc.ResourceNameSingular)),
 		Short:                 dc.ShortDescription,
-		Args:                  cobra.ExactArgs(1),
+		Args:                  cobra.MinimumNArgs(1),
 		ValidArgsFunction:     cmpl.SuggestArgs(cmpl.SuggestCandidatesF(dc.NameSuggestions(s.Client()))),
```

Comment:
I installed this PR locally to verify autocomplete, but noticed that it only autocompletes the first entry. For subsequent autocompletes I only get suggestions for local files. It would be nice to continue to get proper suggestions.